src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -88,10 +114,23 @@
         
         $("#${n}error-message").slideDown(500);
         
-        $('#${n}answer input:radio').click(function (){            
-            $('#submit').removeAttr('disabled');                
-            $('#submit').removeClass('feedback-submit-button');
-            $('#submit').addClass('portlet-form-button');
+        $('#submitFeedbackForm').submit(function (){
+            // disable submit
+            $('input[type=submit]', this).attr('disabled','disabled');
+            $('#${n}submitfeedback').attr('disabled','disabled');
+            $('#${n}submitfeedback').addClass('feedback-submit-button');
+            $('#${n}submitfeedback').removeClass('portlet-form-button');
+            $('#${n}submitfeedback').parent().addClass('ui-disabled');
+        });
+
+       // Enable submitFeedback work here only on Desktop
+       $("#${n}answer input:radio, #${n}answer label").click(function (){
+            if ($('#${n}submitfeedback').attr('disabled')) $('#${n}submitfeedback').removeAttr('disabled');
+            if ($('#${n}submitfeedback').attr('aria-disabled')) $('#${n}submitfeedback').removeAttr('aria-disabled');
+            if ($('#${n}submitfeedback').prop('disabled')) $('#${n}submitfeedback').prop('disabled', false);
+            if($('#${n}submitfeedback').hasClass('feedback-submit-button')) $('#${n}submitfeedback').removeClass('feedback-submit-button');
```

Comment:
spacing after if should be consistent either `if()` or `if ()`.
Also could you wrap the statement in a block, inline if's can be difficult to read.